frontend/src/styles/Team.css

```diff
@@ -33,6 +33,32 @@
     font-family: Arial, Helvetica, sans-serif;
     text-align: left;
   }
+  
+  .btn-group{
+    padding-top: 1vh;
+    width: 100%;
+    justify-content: center;
+  }
+  .btn-group button {
+    background-color: #333; 
+    border: 2px solid #333; 
+    color: white;
+    padding: 5px 10px; 
+    cursor: pointer; 
+    font-size: 1.2vw;
+    font-family: Arial, Helvetica, sans-serif;
+    font-weight: bold;
+    margin-left: 10px;
+  }
+  .btn-group button:hover{
```

Comment:
The styling for the buttons when they are hovered can be changed a little. Right now it is blue text on a white background.

Reply:
If the link is not hovered over, the link color doesn't change so hovering over the button will look like you are just hovering over a white box,

Reply:
I fixed the issue! Thank you for pointing it out.